source/less/blocks/programs.less

```
@@ -22,74 +22,232 @@
   }
 
   @media (min-width: @desktop-width) {
-    flex-direction: row;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
     column-gap: 80px;
     padding: 0 110px;
   }
 }
 
 .porgrams__container {
-  padding: 30px 20px 20px 20px;
+  padding: 20px 20px 22px 20px;
   background-color: @basic_grey;
 
   @media (min-width: @tablet-width) {
-    padding: 77px 52px 53px 52px;
+    position: relative;
+    padding: 77px 52px 56px 52px;
   }
 
   @media (min-width: @desktop-width) {
-    padding: 77px 52px 55px 52px;
+    position: static;
+    padding: 47px 52px 58px 52px;
   }
 }
 
 .porgrams__title {
-  margin: 0 0 29px 0;
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin: 0 0 22px 0;
+  padding-top: 4px;
+  padding-left: 2px;
   font-weight: 400;
   font-size: 24px;
   line-height: 37px;
   text-transform: uppercase;
 
+  &::before {
+    display: block;
+    content: "";
+    width: 50px;
+    height: 50px;
+    margin-right: 7.7%;
+    background-color: @basic_green;
+    border-radius: 50%;
+  }
+
+  &--slim {
+    &::after {
+      position: absolute;
+      top: 0.5px;
+      left: 9px;
+      display: block;
+      content: "";
+      width: 36px;
+      height: 50px;
+      background-image: url("../../img/cat-slim.svg");
```

Неверный адрес у изображений, после сборки твой файл стилей будет просто в папке css.
Проверяй свою работу после сборки.
```suggestion
      background-image: url("../img/cat-slim.svg");
```